Dockerfile

```diff
@@ -58,6 +58,9 @@ RUN pip3 install cget && pip3 install numpy
 # Install rclone
 RUN pip install https://github.com/pfultz2/rclone/archive/master.tar.gz
 
+# Install numpy
+pip install numpy
```

Comment:
It seems this can be removed now?

Reply:
I am not sure for now. I am not sure wether memoryview() function in python2.7 can generate a scalar argument. The current code only generate the shape [1], which is not a scalar and the test failed in Debug build. So I need further investigation on it.

Reply:
If you look at the unit test, I still use the numpy to generate a scalar. We have the line pip3 install numpy for python version 3.x. But we do not have that for Python2.7, so I add this line here. Actually, I am not sure whether it works.

Reply:
Can you use `create_buffer` to create the scalar instead? If the test is broken on python 2.7 then just skip it on python 2.

I would prefer not to require a dependency directly on numpy since it may not be installed(and wont get installed with rbuild).